code/game/objects/items/weapons/storage/vendor_loot_boxes.dm

```diff
@@ -206,4 +206,5 @@
 		new /obj/random/common_oddities/always_spawn(src)
 		new /obj/random/common_oddities/always_spawn(src)
 		new /obj/random/common_oddities/always_spawn(src)
+		new /obj/random/common_oddities/always_spawn(src)
```

Comment:
you need to change the box desc for this one

Reply:
I have changed it to Quadruple now